CacheMoney/backend/pom.xml

```diff
@@ -51,6 +61,12 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>com.microsoft.sqlserver</groupId>
+			<artifactId>mssql-jdbc</artifactId>
+			<version>10.2.0.jre11</version>
```

Comment:
I think this might be an issue. Since our project is requiring Java 8 but it looks like this is requiring Java 11. I may be wrong about this so feel free to correct me. I got this working locally with this dependency instead.
<dependency>
		<groupId>com.microsoft.sqlserver</groupId>
		<artifactId>mssql-jdbc</artifactId>
		<version>7.4.1.jre8</version>
</dependency>

Reply:
Addressed